### ISSUE TITLE
Cannot send embeds from Linux

### ISSUE THREAD

**Issue by lSh4dowl** [lovvskillz/python-discord-webhook#96]
Hello, when I try to send the standard embed example:

```Python
from discord_webhook import DiscordWebhook, DiscordEmbed

webhook = DiscordWebhook(url='your webhook url')

# create embed object for webhook
# you can set the color as a decimal (color=242424) or hex (color='03b2f8') number
embed = DiscordEmbed(title='Your Title', description='Lorem ipsum dolor sit', color='03b2f8')

# add embed object to webhook
webhook.add_embed(embed)

response = webhook.execute()
```

from my Raspberrypi3, I get the following Error:
`[1/1] Webhook status code 400: {"embeds": ["0"]}`

On Macos the same Code works fine.

**Comment by lSh4dowl** [comment]
Update:

I found what's causing this.
For some Reason, the embed Objects variable `embed.color` on  Linux has the Value `9d13bf` and on MacOS it has the value `10294207`.

So it just needs to be converted from Hex to Dec for it to work on Linux.

**Comment by OmarSaidIbrahim** [comment]
Ive literally spent the past 4 hours to understand why it was working on my Raspberry Pi (Linux) but not on my Mac (MacOS).

Life Saver 🛟

**Comment by thebetauser** [comment]
This is actually a bug introduced in 0.16. I spent about an hour tracking this down. 

The type hint added [here](https://github.com/lovvskillz/python-discord-webhook/blob/08c4e62d41995e2db99e65bce02b8d78f308896d/discord_webhook/webhook.py#L69) casts a str to int however this fails since it's not possible to do int(str) of a string containing hex digits. 

The previous version converted it correctly via
`        self.color = int(color, 16) if isinstance(color, str) else color
`

going forward if this is not changed then the official documents should be updated to note that the color parameter must be an int and can no longer be a string

**Comment by lovvskillz** [comment]
should be fixed in 0.16.1
